lib/Conversion/HandshakeToFIRRTL/HandshakeToFIRRTL.cpp

```diff
@@ -670,17 +699,33 @@ static void buildConstantLogic(handshake::ConstantOp *oldOp,
       createConstantOp(constantType, constantValue, insertLoc, rewriter));
 }
 
+static void buildBufferLogic(handshake::BufferOp *oldOp,
+                             ValueVectorList portList, Location insertLoc,
+                             ConversionPatternRewriter &rewriter) {
+  ValueVector inputSubfields = portList[0];
+  Value inputValid = inputSubfields[0];
+  Value inputReady = inputSubfields[1];
+
+  ValueVector outputSubfields = portList[1];
+  Value outputValid = outputSubfields[0];
+  Value outputReady = outputSubfields[1];
+
+  Value clock = portList[2][0];
+  Value reset = portList[3][0];
+}
```

Comment:
Seems like you're missing something here?

Reply:
Yes, the logic is not implemented in this patch. The buffer sub-module is an empty placeholder.

Reply:
Like I mentioned in the patch description, since the buffer here can be implemented separately as an IP, maybe that is a more convenient way.